README.md

```diff
@@ -49,3 +49,36 @@ To remove the created objects just:
 ```shell
 make app-delete
 ```
+
+## Installing and using the template
+
+An Openshift template is provided that let you deploy the current workload state
+by using some parameters; It is required to create the rbac objects at `deploy/admin`
+directory by:
+
+```shell
+kustomize build deploy/admin | oc create -f -
+```
+
+Afterward you can just do the below for installing the template:
+
+```shell
+make template-create
```

Comment:
This steps fails without the image base:

```
Makefile:58: *** IMG_BASE is empty; export IMG_BASE=quay.io/namespace.  Stop.
```

Reply:
Thanks mate! I have updated the error messages in the Makefile for being more descriptive and I have updated the example in the documentation.